src/BlueberryStaking.sol

```diff
@@ -377,8 +376,8 @@ contract BlueberryStaking is
             revert InvalidLength();
         }
 
-        // lockdrop period must be complete i.e 2 months
-        if (block.timestamp <= deployedAt + 60 days) {
+        // lockdrop period must be complete i.e 1 month
+        if (block.timestamp <= deployedAt + 30 days) {
```

Comment:
For future proofing, it would be better to have a constant defined like so:

```solidity
uint256 public constant LOCKDROP_DURATION = 30 days;
```

Use that constant here, and then use `LOCKDROP_DURATION / 2` in `getPrice()`.

Reply:
Agreed this is a better choice.